src/app/tl-footer/tl-footer.component.scss

```diff
@@ -8,6 +8,9 @@
     --wvr-font-family-sans-serif: var(--tl-font-family-sans-serif);
     --wvr-nav-link-font-size: var(--tl-nav-link-font-size);
     --footer-height: 50px;
+    .navbar {
```

Comment:
Lets remove this and solve it in the context of weaver

Reply:
https://github.com/orgs/TAMULib/projects/4#card-37110974

Reply:
Added a PR - https://github.com/TAMULib/weaver-components/pull/90 - to address wvr-footer navbar class to have height of 100%